js/hangman.js

```diff
@@ -0,0 +1,129 @@
+let pass = "Bez pracy nie ma kołaczy";
```

Comment:
This code needs to be rewriten. A lot of spaghetti code here.

Basic changes which shall be applied:
1) Create separated module (to avoid global scope pollution), executed once DOM is being loaded
2) Split the code on the more functions - each function per functionality, ex. add function render for rendering content in DOM
3) You can define Hangman class and define appropriate functions inside